components/dashboard/kpi-grid.tsx

```
@@ -0,0 +1,77 @@
+  ];
+  // Secondary chips (Tier 2)
+  const secondary = [
+    { key: 'Members', value: loading || !data ? <Skeleton className="h-4 w-6" /> : data.members },
+    { key: 'Total Received', value: loading || !data ? <Skeleton className="h-4 w-12" /> : formatAmount(data.totalInflow, { currency: tokenSymbol, compact: true }) },
+    { key: 'Top Holder', value: loading || !data ? <Skeleton className="h-4 w-10" /> : formatPercent(data.topHolderPct ?? 0, { digits: 1 }) },
+  ];
+  return (
+    <section className="flex flex-col gap-4" aria-label="Overview">
+      <div className="flex items-center justify-between px-1">
+        <h2 className="eyebrow">Overview</h2>
+        <div className="text-[11px] text-[var(--text-color-muted)]" role="status">{data ? <span className={freshness.isStale ? "text-amber-500" : undefined}>Updated {freshness.ageSeconds}s ago</span> : <Skeleton className="h-3 w-24" />}</div>
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)